cmd/mysterium_client/command_run/command.go

```diff
@@ -51,15 +52,23 @@ func NewCommandWith(
 
 	httpApiServer := tequilapi.NewServer(options.TequilaApiAddress, options.TequilaApiPort, router)
 
-	return &CommandRun{
+	cmd := &CommandRun{
 		connectionManager,
 		httpApiServer,
+		nil,
```

Comment:
Weird. You could construct 2 different commands depending on options:
- `command_run` - 
- `command_interactive` - which starts `command_run` thru composition